src/contracts/GPv2Settlement.sol

```diff
@@ -51,6 +52,43 @@ contract GPv2Settlement {
         uniswapFactory = uniswapFactory_;
     }
 
+    /// @dev Execute a Uniswap trade exchanging the given amount of tokenIn
```

Comment:
ubernit: As an internal function, I believe this should go further down in the contract: <https://solidity.readthedocs.io/en/v0.7.4/style-guide.html#order-of-functions>